frontend/src/components/SignInUpModal.js

```diff
@@ -17,16 +17,22 @@ export default class LogInUp extends Component {
       error: null,
       initialTab: 'login',
       recoverPasswordSuccess: null,
+      credentials: null,
     };
   }
 
-  async onLogin() {
+  onLogin() {
     const email = document.querySelector('#email').value;
     const password = document.querySelector('#password').value;
 
     //sign in the user
     firebase.auth().signInWithEmailAndPassword(email, password)
-        .then ( response => this.onLoginSuccess())
+        .then ( response => {
+          this.setState({
+            credentials: response.user
+          })
+          this.onLoginSuccess()}
+              )
```

Comment:
Looks kinda wonky here. Maybe put the paranthesis on line 35 to line 34 as well as the ".catch(function(error) {" to line 34.
`this.onLoginSuccess()}).catch(function(error) {`